qa/vagrant/src/main/java/org/elasticsearch/packaging/util/Archives.java

```diff
@@ -255,7 +253,11 @@ private static void verifyDefaultInstallation(Installation es, Distribution dist
         ).forEach(configFile -> assertThat(es.config(configFile), file(File, owner, owner, p660)));
     }
 
-    public static void runElasticsearch(Installation installation, Shell sh) throws IOException {
+    public static void runElasticsearch(Installation installation) throws Exception {
+        runElasticsearch(installation, new Shell());
```

Comment:
I just removed this recently. We need to always pass in a Shell, so we can control when/if JAVA_HOME is set. Please revert this part.